configs/openmrs_config/concepts/past_medical_history.csv

```diff
@@ -7,6 +7,13 @@ c311aec0-dd3d-4c84-a4ab-601fab82ca37,,,Antihypertensive medication use,បច្
 157621ff-16ac-4dd4-bc96-6164d3248807,,,Premature heart disease/stroke in first degree relatives,ធ្លាប់មានប្រវត្តិជំងឺបេះដូង ឬគ្រោះថ្នាក់សរសៃឈាមខួរក្បាលចំពោះសាច់ញាតិបង្កើត  (ខ្សែទីមួយ),Premature heart disease/stroke in first degree relatives,ធ្លាប់មានប្រវត្តិជំងឺបេះដូង ឬគ្រោះថ្នាក់សរសៃឈាមខួរក្បាលចំពោះសាច់ញាតិបង្កើត  (ខ្សែទីមួយ),,,Question,Coded,Yes;No,,,
 ad8f9ff4-63f1-40b9-b405-aef4ae674c2d,,,Past Medical History,ប្រវត្តិសុខភាពពីអតីតកាល,Past Medical History,ប្រវត្តិសុខភាពពីអតីតកាល,,,ConvSet,N/A,,Known hypertensive;Known diabetic;Antihypertensive medication use;Antidiabetic medication use;Diabetes/Kidney disease in first degree relatives;Premature heart disease/stroke in first degree relatives,,
 e0fffb24-5973-40a9-af11-2e4033831747,,,Presence of TB symptoms,មានរោគសញ្ញាសង្ស័យរបេង,Presence of TB symptoms,មានរោគសញ្ញាសង្ស័យរបេង,,,Question,Coded,Yes;No,,,
+b10eae79-8358-4653-822b-33336644db36,,,Suspected TB symptom,រោគសញ្ញាសង្ស័យរបេង,Suspected TB symptom,រោគសញ្ញាសង្ស័យរបេង,,,Question,Coded,Yes;No,,,
+aef248da-8183-4a47-ba60-28e9617dff95,,,Cough,ក្អក,Cough,ក្អក,,,Misc,N/A,,,,
+140238AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,,Fever,ក្តៅខ្លួន,Fever,ក្តៅខ្លួន,,,Misc,N/A,,,,
+53e7d24b-20e1-4e54-a5ec-e338d48aaff0,,,Nighttime sweat,បែកញើសពេលយប់,Nighttime sweat,បែកញើសពេលយប់,,,Misc,N/A,,,,
+85231ffc-1014-466c-8627-fc21ac7f30f4,,,Lost weight,ស្រកទំងន់,Lost weight,ស្រកទំងន់,,,Misc,N/A,,,,
+f7a59dd0-127b-489f-aa6d-9723f18c2220,,,TB symptoms,រោគសញ្ញាជំងឺរបេង,TB symptoms,រោគសញ្ញាជំងឺរបេង,,,Question,Coded,Cough;Fever;Nighttime sweat;Lost weight,,,
```

Comment:
Why not reuse the existing TB symptoms?

https://github.com/openmrs/ozone-distro-cambodia/blob/main/configs/openmrs_config/concepts/past_medical_history.csv#L44